dev/ci/pnpm-install-with-retry.sh

```diff
@@ -10,7 +10,7 @@ while : ; do
     echo "--- ✂️ timeout reached, aborting".
     exit 1
   fi
-  if yarn --immutable --network-timeout 30000 "$@" 2> >(tee "$tmp_log">&2); then
+  if pnpm install --fetch-timeout 30000 --silent "$@" 2> >(tee "$tmp_log">&2); then
```

Comment:
Shouldn't CI scripts run `install` with `--frozen-lockfile`?

```suggestion
  if pnpm install --frozen-lockfile --fetch-timeout 30000 --silent "$@" 2> >(tee "$tmp_log">&2); then
```